src/event-listeners/index.js

```diff
@@ -93,6 +149,9 @@ export default function setupEventListeners(store, canvas) {
       case 'ShiftRight':
         store.dispatch(unsetShiftKeyDown())
         break
+      case 'ControlLeft':
```

Comment:
```suggestion
      case 'ControlLeft':
      case 'ControlRight':
```

we should handle both keyboard keys, just like with ShiftLeft and ShiftRight, just above this ^^